### ISSUE TITLE
pylama is complaining about pylint not existing

### ISSUE THREAD

**Issue by rzuckerm** [PyCQA/pylint#4399]
### Steps to reproduce

```
$ virtualenv -p python3 pylama
...
$ . pylama/bin/activate
(pylama) $ pip install pylama pylint
(pylama) $ pylama -l pylint
...
```

### Current behavior

```
(pylama) $ pylama -l pylint
WARNING:root:Linter `pylint` not found.
```
```python
(pylama) $ python3
Python 3.8.8 (default, Feb 20 2021, 21:09:14) 
[GCC 7.5.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from pylama.lint.pylama_pylint import Linter
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/tmp/master-ouno23vh/test_pkg/.porg/venvs/3.8/lib/python3.8/site-packages/pylama/lint/pylama_pylint.py", line 7, in <module>
    from pylint.__pkginfo__ import numversion
ImportError: cannot import name 'numversion' from 'pylint.__pkginfo__' (/tmp/master-ouno23vh/test_pkg/.porg/venvs/3.8/lib/python3.8/site-packages/pylint/__pkginfo__.py)
>>> 
```

### Expected behavior

Prior to 2.8.0, `pylama` worked just fine. I think `pylama` is expecting `numversion` to have a `tuple` containing the major, minor, and patch version digits. In 2.7.4, there was [this](https://github.com/PyCQA/pylint/blob/pylint-2.7.4/pylint/__pkginfo__.py#L34). In 2.8.0, that variable seems to have been removed.

### pylint --version output

Result of `pylint --version` output:

```console
(pylama) $ pylint --version
pylint 2.8.0
astroid 2.5.5
Python 3.8.8 (default, Feb 20 2021, 21:09:14) 
[GCC 7.5.0]
```

Other packages installed:

```console
(pylama) $ pip freeze
astroid==2.5.5
isort==5.8.0
lazy-object-proxy==1.6.0
mccabe==0.6.1
pycodestyle==2.7.0
pydocstyle==6.0.0
pyflakes==2.3.1
pylama==7.7.1
pylint==2.8.0
snowballstemmer==2.1.0
toml==0.10.2
wrapt==1.12.1
```

**Comment by rzuckerm** [comment]
Although this is probably a `pylama` bug, that project hasn't released a new version since 2019, so it looks like it is an abandoned projected. However, our CI uses `pylama`, and due to this problem, I cannot upgrade `pylint` to the latest version.

**Comment by Pierre-Sassoulas** [comment]
Sorry about that, the import should be changed to ``from pylint import __version__`` line 7 in ``pylama/lint/pylama_pylint.py``. Maybe we could make this retro compatible if a new release of pylama is unlikely.

**Comment by Pierre-Sassoulas** [comment]
I opened a PR in pylama, let's hope it get merged. We're not going to add back this (we might want to add ``scm_setuptools`` later, upgrading manually the version do not make sense), the real metadata of the package are accessible with:
```python
import importlib
metadata.metadata('pylint')
````
(Could be a better fix than what I did for pylama.)

**Comment by Pierre-Sassoulas** [comment]
Actually we can stay compatible.

**Comment by rzuckerm** [comment]
@Pierre-Sassoulas Thanks, for you quick response on this issue.
